single-line-diagram-core/src/main/resources/topologicalBaseVoltages.css

```diff
@@ -1,72 +1,71 @@
+.sld-vl120to180-8 {--sld-vl-color: #2CBD94; stroke:var(--sld-vl-color)}
+.sld-vl120to180-9 {--sld-vl-color: #00839E; stroke:var(--sld-vl-color)}
+.sld-vl70to120-0 {--sld-vl-color: #CC5500; stroke:var(--sld-vl-color)}
+.sld-vl70to120-1 {--sld-vl-color: #4A6600; stroke:var(--sld-vl-color)}
+.sld-vl70to120-2 {--sld-vl-color: #EFB2DD; stroke:var(--sld-vl-color)}
+.sld-vl70to120-3 {--sld-vl-color: #6E7A00; stroke:var(--sld-vl-color)}
+.sld-vl70to120-4 {--sld-vl-color: #E685AE; stroke:var(--sld-vl-color)}
+.sld-vl70to120-5 {--sld-vl-color: #8E8400; stroke:var(--sld-vl-color)}
+.sld-vl70to120-6 {--sld-vl-color: #DD596B; stroke:var(--sld-vl-color)}
+.sld-vl70to120-7 {--sld-vl-color: #A37B00; stroke:var(--sld-vl-color)}
+.sld-vl70to120-8 {--sld-vl-color: #D4432C; stroke:var(--sld-vl-color)}
+.sld-vl70to120-9 {--sld-vl-color: #B76B00; stroke:var(--sld-vl-color)}
+.sld-vl50to70-0 {--sld-vl-color: #A020EF; stroke:var(--sld-vl-color)}
+.sld-vl50to70-1 {--sld-vl-color: #7F0848; stroke:var(--sld-vl-color)}
+.sld-vl50to70-2 {--sld-vl-color: #B7DBFE; stroke:var(--sld-vl-color)}
+.sld-vl50to70-3 {--sld-vl-color: #960C6D; stroke:var(--sld-vl-color)}
+.sld-vl50to70-4 {--sld-vl-color: #8DA6FE; stroke:var(--sld-vl-color)}
+.sld-vl50to70-5 {--sld-vl-color: #AD109A; stroke:var(--sld-vl-color)}
+.sld-vl50to70-6 {--sld-vl-color: #6F66FB; stroke:var(--sld-vl-color)}
+.sld-vl50to70-7 {--sld-vl-color: #BC14C4; stroke:var(--sld-vl-color)}
+.sld-vl50to70-8 {--sld-vl-color: #7F42F6; stroke:var(--sld-vl-color)}
+.sld-vl50to70-9 {--sld-vl-color: #B11AD9; stroke:var(--sld-vl-color)}
+.sld-vl30to50-0 {--sld-vl-color: #FF8290; stroke:var(--sld-vl-color)}
+.sld-vl30to50-1 {--sld-vl-color: #7F6F41; stroke:var(--sld-vl-color)}
+.sld-vl30to50-2 {--sld-vl-color: #F6D9FF; stroke:var(--sld-vl-color)}
+.sld-vl30to50-3 {--sld-vl-color: #99784E; stroke:var(--sld-vl-color)}
+.sld-vl30to50-4 {--sld-vl-color: #FFC3FB; stroke:var(--sld-vl-color)}
+.sld-vl30to50-5 {--sld-vl-color: #B27D5B; stroke:var(--sld-vl-color)}
+.sld-vl30to50-6 {--sld-vl-color: #FFADE3; stroke:var(--sld-vl-color)}
+.sld-vl30to50-7 {--sld-vl-color: #CC7E68; stroke:var(--sld-vl-color)}
+.sld-vl30to50-8 {--sld-vl-color: #FF97BF; stroke:var(--sld-vl-color)}
+.sld-vl30to50-9 {--sld-vl-color: #E57B75; stroke:var(--sld-vl-color)}
+.sld-vl0to30-0 {--sld-vl-color: #AAAE27; stroke:var(--sld-vl-color)}
+.sld-vl0to30-1 {--sld-vl-color: #195B0F; stroke:var(--sld-vl-color)}
+.sld-vl0to30-2 {--sld-vl-color: #EABABE; stroke:var(--sld-vl-color)}
+.sld-vl0to30-3 {--sld-vl-color: #2D6C13; stroke:var(--sld-vl-color)}
+.sld-vl0to30-4 {--sld-vl-color: #DDA193; stroke:var(--sld-vl-color)}
+.sld-vl0to30-5 {--sld-vl-color: #477D17; stroke:var(--sld-vl-color)}
+.sld-vl0to30-6 {--sld-vl-color: #CE9A6E; stroke:var(--sld-vl-color)}
+.sld-vl0to30-7 {--sld-vl-color: #648D1C; stroke:var(--sld-vl-color)}
+.sld-vl0to30-8 {--sld-vl-color: #BEA04A; stroke:var(--sld-vl-color)}
+.sld-vl0to30-9 {--sld-vl-color: #869D22; stroke:var(--sld-vl-color)}
```

Comment:
Like I said in baseVoltages.css I'd prefer to put the stroke in the `components.css` file